overlays/python38/Pipfile.lock

```diff
@@ -670,7 +670,7 @@
                 "sha256:aaf6ab2fcbc68ab6c4c6ba924b57991a388ccc1c8fae782f2433340a40743dfe",
                 "sha256:cf7bae999005dce012738d09610e69a47acd765797c0365c803f96c21ea2f32b"
             ],
-            "markers": "python_version >= '3.4' and python_version < '4'",
+            "markers": "python_version >= '3.4' and python_full_version < '4.0.0'",
```

Comment:
Why does this use full version, but we change the (other lockfile)[https://github.com/thoth-station/s2i-minimal-notebook/pull/532/files#diff-b93e76970a00833f87a2cbc331328c72129a6b977ef77567eafbe377338613b3R649] to move away from full versson?

Reply:
i m not sure on why , if it feel critical , lets file an issue and address it there.

Reply:
Doesn't feel critical, we can go ahead and move forward. Has testing bee done with these changes?

Reply:
yes i tested it on my system